layers/+window-management/exwm/funcs.el

```diff
@@ -66,8 +66,8 @@
   (setq exwm--toggle-workspace exwm-workspace-current-index))
 
 (defun exwm/exwm-app-launcher ()
-  "Launches an application in your PATH.
-Can show completions at point for COMMAND using helm"
+  "Launch an application in PATH.
+Optionally, this has helm integration when a helm package is enabled."
```

Comment:
"a helm package" -> "the helm package" or "helm package" or simply "helm".

helm is a proper noun here.